### PR TITLE
feat(conf): bonita-platform-init-community.properties removed

### PR DESCRIPTION
**Pull request by tbouffard**
Closes to [BS-17105](https://bonitasoft.atlassian.net/browse/BS-17105)

### CODE REVIEW

**Comment by educhastenier** [comment]
You're missing the other pages where we urge Sam to update the file

**Comment by tbouffard** [comment]
@educhastenier I guess what you are talking about has already been done with PR #279

**Comment by educhastenier** [comment]
yes indeed !

File: md/release-notes.md [hunk 1]
```diff
@@ -42,6 +42,16 @@ To be documented
 
 ### API removals
 
+### Configuration changes
+
+#### Removal of the ```platform_init_engine/bonita-platform-init-community.properties``` file
+
+In previous versions, this file should be updated to activate the archive configurability feature (Performance edition).  
+As it is now activated by default when using the Performance edition, the file has been removed.
```
Comment (educhastenier) [inline md/release-notes.md]: This simplifies the configuration.

**Comment by educhastenier** [comment]
Looks good to me :+1:

**Pull request merged**